R/process_content.R

```diff
@@ -0,0 +1,14 @@
+process_content <- function(gitai, content) {
```

Comment:
I thought if it is not good idea to add some small descriptions together with functions (for roxygenizeing it in future) - this could help also developer when reviewing.

Reply:
Sure, for exported functions definitely, optionally for non-exported.
I usually do that when I have a more stable interface, right now we are changing and experimenting a lot, so don't want to waste time on docs that will be outdated soon, but yeah we will do that.